SPECS/dotnet-runtime-3.1/dotnet-runtime-3.1.spec

```diff
@@ -1,96 +1,81 @@
 Summary:        Microsoft .NET Core Runtime
 Name:           dotnet-runtime-3.1
 Version:        3.1.5
-Release:        1%{?dist}
+Release:        2%{?dist}
+License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-License:        MIT
-Url:            https://github.com/dotnet/core
 Group:          Development/Tools
-ExclusiveArch:  x86_64
+URL:            https://github.com/dotnet/core
 Source0:        https://download.visualstudio.microsoft.com/download/pr/d00eaeea-6d7b-4e73-9d96-c0234ed3b665/0d25d9d1aeaebdeef01d15370d5cd22b/dotnet-runtime-3.1.5-linux-x64.tar.gz
-
+Requires:       glibc
 Requires:       icu
-Requires:       libunwind
 Requires:       krb5
+Requires:       libgcc
+Requires:       libstdc++
+Requires:       libunwind
 Requires:       lttng-ust
 Requires:       openssl
 Requires:       zlib
-Requires:       glibc
-Requires:       libgcc
-Requires:       libstdc++
+ExclusiveArch:  x86_64
 
 %description
 .NET Core is a development platform that you can use to build command-line
 applications, microservices and modern websites.
 
 %prep
-%setup -qc dotnet-runtime-%{version}
+%setup -qc -T -a 0 dotnet-runtime-%{version}
 
 %build
 
 %install
 mkdir -p %{buildroot}%{_libdir}/dotnet
 mkdir -p %{buildroot}%{_docdir}/dotnet-runtime-%{version}
-cp LICENSE.txt ThirdPartyNotices.txt %{buildroot}%{_docdir}/dotnet-runtime-%{version}
-rm LICENSE.txt ThirdPartyNotices.txt
+
 cp -r * %{buildroot}%{_libdir}/dotnet
+rm %{buildroot}%{_libdir}/dotnet/LICENSE.txt %{buildroot}%{_libdir}/dotnet/ThirdPartyNotices.txt
```

Comment:
Please don't use macros for removing files local to the package. The macros reflect the paths on the final system, so they are a different thing. Thomas is working on getting the linter to be slightly less ambitious here.:)